client/js/middleware/serialization/qbank/serializers/base.js

```diff
@@ -0,0 +1,11 @@
+import genusTypes       from '../../../../constants/genus_types';
+
+export default function serializeQuestion(originalItem, newAttributes) {
```

Comment:
here you call 2nd param newAttributes, then later call it with question and also newQuestionAttributes.. what should it be?